include/soci/type-conversion-traits.h

```diff
@@ -36,6 +36,29 @@ struct type_conversion
     }
 };
 
+// Most majority of database engines ignore unsigned integer related types.
```

Comment:
Some typos: "most majority" is redundant, "trait" is probably supposed to be "handle".